test/Benchmarks/Metrics/MetricsViewBenchmarks.cs

```diff
@@ -31,11 +32,12 @@
   DefaultJob : .NET 6.0.10 (6.0.1022.47605), X64 RyuJIT AVX2
 
 
-|                                   Method |        ViewConfig |     Mean |   Error |  StdDev | Allocated |
-|----------------------------------------- |------------------ |---------:|--------:|--------:|----------:|
-| CounterMeasurementRecordingWithThreeTags |            NoView | 188.1 ns | 1.73 ns | 1.54 ns |         - |
-| CounterMeasurementRecordingWithThreeTags | ViewNoInstrSelect | 193.3 ns | 0.66 ns | 0.61 ns |         - |
-| CounterMeasurementRecordingWithThreeTags |  ViewSelectsInstr | 271.3 ns | 2.18 ns | 1.94 ns |         - |
```

Comment:
this ~80 ns overhead is now reduced to ~15ns. Users can now use View to drop unwanted dimension (saving backend storage costs), by paying a minimal cost at Sdk side.